mkdocs.yml

```diff
@@ -10,72 +10,114 @@ edit_uri: blob/main/docs/
-      name: Switch to light mode
+    - scheme: default
+      media: "(prefers-color-scheme: light)"
+      toggle:
+        icon: material/brightness-7
+        name: Switch to dark mode
+    - scheme: slate
+      media: "(prefers-color-scheme: dark)"
+      toggle:
+        icon: material/brightness-4
+        name: Switch to light mode
   features:
-  - content.action.edit
-  - content.code.copy
-  - content.code.select
-  - navigation.instant
-  - navigation.tracking
-  - navigation.tabs
-  - navigation.tabs.sticky
-  - navigation.top
-  - search.highlight
-  - search.share
-  - toc.integrate
+    - content.action.edit
+    - content.code.copy
+    - content.code.select
+    - navigation.instant
+    - navigation.tracking
+    - navigation.tabs
+    - navigation.tabs.sticky
+    - navigation.top
+    - search.highlight
+    - search.share
+    - toc.integrate
 
 extra:
   version:
     provider: mike
   social:
-  - icon: fontawesome/brands/slack
-    link: https://dependencytrack.org/slack
-  - icon: fontawesome/brands/github
-    link: https://github.com/DependencyTrack
-  - icon: fontawesome/brands/docker
-    link: https://hub.docker.com/u/dependencytrack
-  - icon: fontawesome/brands/twitter
-    link: https://twitter.com/DependencyTrack
+    - icon: fontawesome/brands/slack
+      link: https://dependencytrack.org/slack
+    - icon: fontawesome/brands/github
+      link: https://github.com/DependencyTrack
+    - icon: fontawesome/brands/docker
+      link: https://hub.docker.com/u/dependencytrack
+    - icon: fontawesome/brands/twitter
+      link: https://twitter.com/DependencyTrack
 
 markdown_extensions:
-- attr_list
-- admonition
-- footnotes
-- md_in_html
-- pymdownx.details
-- pymdownx.emoji:
-    emoji_index: !!python/name:material.extensions.emoji.twemoji
-    emoji_generator: !!python/name:material.extensions.emoji.to_svg
-- pymdownx.highlight:
-    anchor_linenums: true
-    line_spans: __span
-    pygments_lang_class: true
-- pymdownx.inlinehilite
-- pymdownx.snippets
-- pymdownx.superfences:
-    custom_fences:
-    - name: mermaid
-      class: mermaid
-      format: !!python/name:pymdownx.superfences.fence_code_format
-- tables
-- toc:
-    permalink: true
-    toc_depth: 3
+  - attr_list
+  - admonition
+  - footnotes
+  - md_in_html
+  - pymdownx.details
+  - pymdownx.emoji:
+      emoji_index: !!python/name:material.extensions.emoji.twemoji
+      emoji_generator: !!python/name:material.extensions.emoji.to_svg
+  - pymdownx.highlight:
+      anchor_linenums: true
+      line_spans: __span
+      pygments_lang_class: true
+  - pymdownx.inlinehilite
+  - pymdownx.snippets
+  - pymdownx.superfences:
+      custom_fences:
+        - name: mermaid
+          class: mermaid
+          format: !!python/name:pymdownx.superfences.fence_code_format
+  - tables
+  - toc:
+      permalink: true
+      toc_depth: 3
 
 plugins:
-- search
-- social
+  - search
+  - social
 
 nav:
+<<<<<<< jm-auth-and-roles-adr
```

Comment:
There are some merge conflict artifacts in here. Can you resolve them, and also revert all the whitespace changes please?